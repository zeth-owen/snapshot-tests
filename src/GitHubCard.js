import React from 'react';
import Card from 'react-bootstrap/Card';
import Button from 'react-bootstrap/Button';
import zeth from './zeth.jpg'

export default function GitHubCard() {
  return (
    <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src={zeth} alt="Zeth" style={{ width: '100%', height: 'auto' }} />
      <Card.Body>
        <Card.Title>zeth-owen</Card.Title>
        <Card.Text>
          Heyyy. My name is Zeth, and I am coding right now ðŸ™‚
        </Card.Text>
        <Button variant="primary">Go somewhere</Button>
      </Card.Body>
    </Card>
  );
}

