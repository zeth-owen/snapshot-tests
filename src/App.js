import GitHubCard from './GitHubCard'
import './App.css';

function App() {
  return (
   <GitHubCard />
  );
}

export default App;
