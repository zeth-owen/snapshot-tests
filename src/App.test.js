import renderer from 'react-test-renderer'
import App from './App';
import GitHubCard from './GitHubCard';

test('renders a snapshot', () => {
  const tree = renderer.create(<GitHubCard/>).toJSON()
  console.log(tree)
  expect(tree).toMatchSnapshot()
})

