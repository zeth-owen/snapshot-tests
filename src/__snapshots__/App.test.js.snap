// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    alt="Zeth"
    className="card-img-top"
    src="zeth.jpg"
    style={
      Object {
        "height": "auto",
        "width": "100%",
      }
    }
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      zeth-owen
    </div>
    <p
      className="card-text"
    >
      Heyyy. My name is Zeth, and I am coding right now ðŸ™‚
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
